[Refactoring] 프로젝트 구조 정리

Marcorable opened the pull request:
### 반영사항
- 폴더 구조 정리
- 프로퍼티 정리
- View 구조 정리
- 변수명 변경

### Side effects
- 애니메이션 중 이전 버전과 달리 동작하지 않는 몇가지가 있습니다

Commit:
chore: .DS_Store 파일 ignore

Commit:
chore: 폴더구조 정리

Commit:
fix: 앱 실행 실패 에러 해결 (.task)

Commit:
refactor: View 구조 변경 및 데이터 적용

Commit:
refactor: 불필요한 View 프로퍼티 제거

Commit:
feat: SwiftData 업데이트 추가

Commit:
refactor: 주석추가 및 변수 변경

Guboneui commented on SaveEarth/SaveEarth/Manager/SwiftData/DayInfo.swift:
```diff
@@ -0,0 +1,28 @@
+import SwiftData
+
+@Model
+class DayInfo {
```
`
```suggestion
final class DayInfo {
```

Guboneui commented on SaveEarth/SaveEarth/Manager/SwiftData/Mission.swift:
```diff
@@ -0,0 +1,27 @@
+import SwiftData
+
+@Model
+class Mission {
```
```suggestion
final class Mission {
```

Guboneui commented on SaveEarth/SaveEarth/Manager/SwiftData/TemperatureSwiftDataModel.swift:
```diff
@@ -0,0 +1,27 @@
+import SwiftData
+
+@Model
+class TemperatureSwiftDataModel {
```
```suggestion
final class TemperatureSwiftDataModel {
```

Guboneui commented on SaveEarth/SaveEarth/RootView/ContentView.swift:
```diff
@@ -0,0 +1,191 @@
+    
+    private var dayInfo: DayInfo? { dayInfos.first(where: { $0.date == Date.now.dateDescription }) }
+    
+    @State private var latitude: CGFloat = 76.571640
+    @State private var longitude: CGFloat = -41.666646
+    
+    @State private var isPresentedModal: Bool = false
```
```suggestion
    @State private var latitude: CGFloat = 76.571640
    @State private var longitude: CGFloat = -41.666646
    @State private var isPresentedModal: Bool = false
```

Guboneui commented on SaveEarth/SaveEarth/Manager/SwiftData/DayInfo.swift:
```diff
@@ -0,0 +1,28 @@
+    #Unique<DayInfo>([\.date])
+    let date: String
+    let temperatureData: TemperatureSwiftDataModel
+    var missionList: [Mission]
+    
+    
```
```suggestion
    var missionList: [Mission]

```

Guboneui commented on SaveEarth/SaveEarth/RootView/ContentView.swift:
```diff
@@ -0,0 +1,191 @@
+    
+}
+
+fileprivate extension Date {
+    
+    var dateDescription: String {
+        let dateFormatter = DateFormatter()
+        dateFormatter.dateFormat = "yyyy-MM-dd"
+        return dateFormatter.string(from: self)
+    }
+    
+}
```
```suggestion
fileprivate extension Date {
    
    var dateDescription: String {
        let dateFormatter = DateFormatter()
        dateFormatter.dateFormat = "yyyy-MM-dd"
        return dateFormatter.string(from: self)
    }
}
```

Guboneui commented on SaveEarth/SaveEarth/RootView/ContentView.swift:
```diff
@@ -0,0 +1,191 @@
+        }
+        
+        return indexes.map({ Mission(title: questList[$0]) })
+    }
+    
+}
```
```suggestion
    }
}
```

Guboneui commented on SaveEarth/SaveEarth.xcodeproj/project.pbxproj:
```diff
@@ -193,6 +194,41 @@
 			path = SaveEarthUITests;
 			sourceTree = "<group>";
 		};
+		A9F552D02C398D5700A697A9 /* Manager */ = {
+			isa = PBXGroup;
+			children = (
+				DA5A5ECA2C2FB8CF00955A9A /* Weather */,
+				A59CE1372C2FFD5D00038D84 /* SwiftData */,
+			);
+			path = Manager;
+			sourceTree = "<group>";
+		};
```
데이터 모델들을 포함하는 폴더 이름이 `Manager`로 되어있는데요. `SwiftData`, `Request Model`, `Response Model`을  포함할 수 있도록 `Entities` 라는  이름으로 폴더명을 변경하는 것은 어떨까요?

Marcorable replied:
오.. 너무 좋습니다

Guboneui reviewed (commented):
머지될 코드라고  생각되어 눈에 보이는  사소한 컨벤션들 `Suggestion`으로  남겨놨습니다! 보시고 의견 주세용~

Marcorable committed:
Update SaveEarth/SaveEarth/Manager/SwiftData/DayInfo.swift

Co-authored-by: 보니 <73548875+Guboneui@users.noreply.github.com>

Marcorable committed:
Update SaveEarth/SaveEarth/Manager/SwiftData/Mission.swift

Co-authored-by: 보니 <73548875+Guboneui@users.noreply.github.com>

Marcorable committed:
Update SaveEarth/SaveEarth/Manager/SwiftData/TemperatureSwiftDataModel.swift

Co-authored-by: 보니 <73548875+Guboneui@users.noreply.github.com>

Marcorable committed:
Update SaveEarth/SaveEarth/RootView/ContentView.swift

Co-authored-by: 보니 <73548875+Guboneui@users.noreply.github.com>

Marcorable committed:
Update SaveEarth/SaveEarth/Manager/SwiftData/DayInfo.swift

Co-authored-by: 보니 <73548875+Guboneui@users.noreply.github.com>

Marcorable committed:
Update SaveEarth/SaveEarth/RootView/ContentView.swift

Co-authored-by: 보니 <73548875+Guboneui@users.noreply.github.com>

Marcorable committed:
Update SaveEarth/SaveEarth/RootView/ContentView.swift

Co-authored-by: 보니 <73548875+Guboneui@users.noreply.github.com>

Commit:
chore: 폴더 구조 변경

- Source/Resource 상위 구조 생성
- Entities 상위 그룹 및 RequestModel/ResponseModel 하위 그룹 추가
